inc/az_ulib_ustream_base.h

```diff
@@ -340,11 +340,21 @@
 
 #include "azure/core/_az_cfg_prefix.h"
 
+#ifndef AZ_ULIB_USTREAM_FORWARD_BASE_H
```

Comment:
>AZ_ULIB_USTREAM_FORWARD_BASE_H [](http://example.com/codeflow?start=8&length=30)

Instead of doing this, please move this definition to az_ulib_base.h.